core/templates/pages/about-page/about-page.component.css

```diff
@@ -324,11 +335,12 @@
     width: 100%;
   }
 }
-@media (orientation: landscape) and (max-width: 740px) {
-  .oppia-about-page-section-one,
-  .oppia-about-page-section-one .oppia-about-study-image img {
-    height: 90vh;
```

Comment:
Here, I removed the height because I changed it to 50vh, which is needed to maintain consistency throughout different screen widths.